openmp/libomptarget/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -2913,7 +2913,7 @@ struct AMDGPUPluginTy final : public GenericPluginTy {
   /// Create an AMDGPU plugin and initialize the AMDGPU driver.
   AMDGPUPluginTy()
       : GenericPluginTy(getTripleArch()), Initialized(false),
-        HostDevice(nullptr) {}
+        HostDevice(nullptr), IsAPU(false), OMPX_HSAXnack("HSA_XNACK", false) {}
```

Comment:
TBH I think all these class members can be initialized directly when the class is defined. No need to list them here.